misk/src/main/kotlin/misk/client/ClientMetricsInterceptor.kt

```diff
@@ -36,6 +36,13 @@ class ClientMetricsInterceptor private constructor(
       requestDurationSummary.labels(actionName, "timeout").observe(elapsedMillis)
       requestDurationHistogram.labels(actionName, "timeout").observe(elapsedMillis)
       throw e
+    } catch (e: Exception) {
+      // Something else happened while the connection was in progress and we didn't receive
+      // a complete response. We still want to record any long-running calls, however.
+      val elapsedMillis = stopwatch.stop().elapsed(TimeUnit.MILLISECONDS).toDouble()
+      requestDurationSummary.labels(actionName, "0").observe(elapsedMillis)
```

Comment:
What is the `"0"` label meant to represent?

Reply:
The second label is the response code that can be used to filter time series in signalfx. Since there's no complete response, I chose a code of 0 which is not a valid status code.

Reply:
Given the use of `timeout` in the second label above would it be an option to use a word like `incomplete-response` or something like that?

Reply:
Reading through https://github.com/squareup/cash-signalfx/blob/master/modules/dashboards/rpc/outgoing/main.tf I don't see any reason why that should be a problem.